code/workhorse/update_train_pool.R

```diff
@@ -0,0 +1,36 @@
+source("add_noise.R")
+
+update_train_pool <- function (simulator,select_index) {
+ 
+  add = simulator$setting[["add"]]
+  noise_level = simulator$setting[["noise"]]
+  
+  adddata = simulator$pool[select_index,,drop=FALSE]
```

Comment:
`adddata` is not a good name. Maybe `additional_data` or `new_data`?

Reply:
rename as additional_data.